democritus_dates/dates_and_times.py

```diff
@@ -112,6 +112,9 @@ def date_now(*, convert_to_current_timezone: bool = False, utc: bool = False):
     """
     now = datetime.datetime.now()
 
+    if convert_to_current_timezone and utc :
+        raise Exception("Only One Input Parameter Should Be True!")
```

Comment:
Thanks for working on this and great job!

I have a suggestion. It is a good practice to be as specific as possible when raising errors/exceptions in Python. As such, I recommend changing from `Exception` to  [`ValueError`](https://docs.python.org/3/library/exceptions.html#ValueError) because `ValueError` is more specific and captures the nature of the exception we are trying to communicate.

```suggestion
        raise ValueError("Only One Input Parameter Should Be True!")
```